Source/WebCore/rendering/RenderThemeMac.mm

```diff
@@ -2593,7 +2593,8 @@ static Color titleTextColorForAttachment(const RenderAttachment& attachment, Att
         return nil;
 
     if (auto icon = WebCore::iconForAttachment(fileName, attachmentType, title))
-        return icon->nsImage();
+        return adoptNS([[NSImage alloc] initWithCGImage:icon->image()->platformImage().get() size:NSZeroSize]);
```

Comment:
Is `NSZeroSize the right thing to use here?

Reply:
yes; it means "use the size of the cg image"